scripts/merge-measures-data.js

```diff
@@ -47,9 +47,70 @@ function mergeQpp(qppJson) {
       if (performanceRateInfo.metricType) {
         qppJson[index].metricType = performanceRateInfo.metricType;
       }
+
     }
   });
 
   console.error('did not find measure details for the following', measuresNotFound);
+
+  // separate passes for ecqm data to keep things simple
+  // load extracted ecqm data
+  const generatedEcqmStrataJson = JSON.parse(fs.readFileSync(path.join(__dirname, '../util/generated-ecqm-data.json'), 'utf8'));
```

Comment:
i added some comments to merge-measures-data to make the ecqm additions clearer: https://github.com/CMSgov/qpp-measures-data/pull/32/commits/b3fb1243c5ac53e040e9b2de5b320db2b60e3705